specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-11-15-preview/restorable.json

```diff
@@ -1099,6 +1099,16 @@
               "description": "The operation type of this database event.",
               "$ref": "#/definitions/OperationType"
             },
+            "canUndelete": {
```

Comment:
since the description says it's a `state`, maybe the name should be `xxxStatus`? And it should be an `enum`?